appservice/src/main/java/nu/yona/server/subscriptions/rest/BuddyController.java

```diff
@@ -158,11 +166,11 @@ public BuddyResource(BuddyDTO buddy)
 			}
 
 			HashMap<String, Object> result = new HashMap<String, Object>();
-			result.put(BuddyDTO.USER_REL_NAME,
-					new UserController.UserResourceAssembler(false).toResource(getContent().getUser()));
+			result.put(curieProvider.getNamespacedRelFor(BuddyDTO.USER_REL_NAME),
```

Comment:
Is this necessary? I read in the [documentation](http://docs.spring.io/spring-hateoas/docs/current/reference/html/#spis.curie-provider): "Note that now the prefix (...) automatically appears before all rels which are not registered with IANA, as in ex:orders."